app/uk/gov/hmrc/hecstubs/models/DESResponse.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hecstubs.models
+
+import play.api.libs.json.JsValue
+
+final case class DESResponse(status: Int, responseBody: JsValue)
+
+object DESResponse {
+  val ctutrRes: String =
+    s"""
+       |{
+       |  "CTUTR": {
+       |    "value": "1234567890"
+       |  }
+       |}
+       |""".stripMargin
+
+  val error404Response: String =
```

Comment:
we don't seem to be using the direct `String` values here except to do `Json.parse` on them - perhaps these could all just be of type `JsValue`?

Reply:
we are using them to Jsvalue under Company Profile, yes we can make Jsvalue here